docs/.vuepress/config.js

```diff
@@ -109,17 +109,32 @@ module.exports = {
             {
               title: "Interchain Accounts",
               directory: true,
-              path: "/app_modules",
+              path: "/app-modules",
```

Comment:
Updated to dashed casing to align with other directories and files. E.g. ADRs or `upgrades/developer-guide.md` ..etc